db/migrate/20160420011850_add_post_id_to_comments.rb

```diff
@@ -1,4 +0,0 @@
-class AddPostIdToComments < ActiveRecord::Migration
```

Comment:
This migration should be a reference instead of just an integer
http://edgeguides.rubyonrails.org/active_record_migrations.html#highlighter_896971